### DISCUSSION TITLE
Adjusting "Back to top" to consider the `wheel` event

### DISCUSSION THREAD

**Discussion by jdknight** [pradyunsg/furo#421] [Ideas]
The existing scroll handler logic ties everything to the [`scroll` event](https://developer.mozilla.org/en-US/docs/Web/API/Document/scroll_event). I'm wondering if it would be possible to adjust the "Back to top" implementation to also consider the [`wheel` event](https://developer.mozilla.org/en-US/docs/Web/API/Document/wheel_event) (when applicable), to help avoid rendering the back-to-top overlay when a page is scrolled via navigation events (back/forward; and menu clicks on the sidebar).

**Comment by pradyunsg** [comment]
Seems reasonable to me! Would you be willing to look into implementing this and filing a PR for this?
